src/mds/MDBalancer.cc

```diff
@@ -604,6 +604,8 @@ void MDBalancer::prep_rebalance(int beat)
 	      << dendl;
     }
 
+    mds_meta_load.clear();
```

Comment:
Accidentally left in?

Reply:
no, otherwise meta_load for stopped mds will left there forever